x-pack/plugins/actions/server/builtin_action_types/cases_webhook/service.ts

```diff
@@ -107,17 +104,12 @@ export const createExternalService = (
 
       throwDescriptiveErrorIfResponseIsNotValid({
         res,
-        requiredAttributesToBeInTheResponse: [
-          getIncidentResponseCreatedDateKey,
-          getIncidentResponseExternalTitleKey,
-          getIncidentResponseUpdatedDateKey,
-        ],
+        requiredAttributesToBeInTheResponse: [getIncidentResponseExternalTitleKey],
       });
 
       const title = getObjectValueByKeyAsString(res.data, getIncidentResponseExternalTitleKey)!;
-      const createdAt = getObjectValueByKeyAsString(res.data, getIncidentResponseCreatedDateKey)!;
-      const updatedAt = getObjectValueByKeyAsString(res.data, getIncidentResponseUpdatedDateKey)!;
-      return { id, title, createdAt, updatedAt };
+      const pushedDate = new Date().toISOString();
+      return { id, title, pushedDate };
```

Comment:
It does not make sense to return a `pushedDate` from the `getIncident` because you are fetching a resource that is created and some point in the past but the `pushedDate` is set as the date of the fetching. I think the `pushedDate` should be removed.